src/test/java/com/fryrank/controller/UserMetadataControllerTests.java

```diff
@@ -0,0 +1,44 @@
+package com.fryrank.controller;
+
+import com.fryrank.dal.UserMetadataDAL;
+import com.fryrank.model.UserMetadata;
+import com.fryrank.validator.ValidatorException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static com.fryrank.TestConstants.TEST_ACCOUNT_ID;
+import static com.fryrank.TestConstants.TEST_USERNAME;
+import static com.fryrank.TestConstants.TEST_USER_METADATA_1;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UserMetadataControllerTests {
+    @Mock
+    UserMetadataDAL userMetadataDAL;
+
+    @InjectMocks
+    UserMetadataController userMetadataController;
+
```

Comment:
There should be a test to get user metadata which is the other controller method

Reply:
Done in [e281f82](https://github.com/NickPriv/FryRankBackend/pull/75/commits/e281f825a8834d7e490af094d3f5f49d6a0f9de1).